addons/mail/static/src/core/web/chatter.xml

```diff
@@ -108,6 +111,9 @@
             </t>
         </div>
         <div class="o-mail-Chatter-content">
+            <div class="d-flex p-4" t-if="state.isSearchOpen">
+                <SearchForm onSearch.bind="onSearch" onClear.bind="onClear" searchTerm="state.searchTerm"/>
+            </div>
```

Comment:
With previous comment to pass `useMessageSearch()` hook. This becomes simply:
```xml
<SearchForm searchFeature="messageSearch"/>
```
_(not definitive naming, but you get the idea)_